qiita_pet/handlers/portal.py

```diff
@@ -0,0 +1,41 @@
+from tornado.web import authenticated, HTTPError
```

Comment:
Also, lookng at this, most of the handlers do not have the copyright header. Issue opened: #1321